includes/Shopping/Product.php

```diff
@@ -83,11 +85,7 @@ class Product {
 	 *
 	 * @var array
 	 */
-	protected $aggregate_rating = [
-		'rating_value' => 0.0,
-		'review_count' => 0,
-		'review_url'   => '',
-	];
+	protected $aggregate_rating;
```

Comment:
An "empty" rating array like this is invalid. If provided, there _must_ be a URL as well.

So if there's no rating information, this needs to be omitted.